docs/whats-new.asciidoc

```diff
@@ -4,172 +4,36 @@
-
-[role="screenshot"]
-image::whats-new/images/8.16/knowledge-base-add-index-config.png[Knowledge Base's Edit index entry menu,80%]
-
-[float]
-== Entity Analytics enhancements
-
-[float]
-=== Manage persisted entity metadata with entity store
-
-preview:[] The {security-guide}/entity-store.html[entity store] feature allows you to query, reconcile, and maintain entity metadata from various sources, such as ingested logs, integrated identity providers, external asset repositories, and more. By extracting and storing entities from all indices in the {elastic-sec} default data view, the entity store lets you query entity metadata without real-time data searches.
-
-After you enable the entity store, the Entity Analytics dashboard displays the {security-guide}/detection-entity-dashboard.html#entity-entities[**Entities** section], which offers a comprehensive view of all hosts and users in your environment. You can filter them by their source, entity risk level, and asset criticality level.
-
-[role="screenshot"]
-image::whats-new/images/8.16/entities-section.png[Entities section of the Entity Analytics dashboard]
-
-[float]
-=== Asset criticality is available by default
-
-The advanced setting for enabling {security-guide}/asset-criticality.html[asset criticality] has been removed, and this feature is now available by default.
-
-[float]
-=== Run entity risk scoring in multiple spaces 
-
-You can now enable and run {security-guide}/entity-risk-scoring.html[entity risk scoring] in multiple {kib} spaces. This allows you to analyze and monitor entity risk in different contexts simultaneously.
-
-[float]
-=== Recalculate entity risk scores after file upload
-
-When you {security-guide}/asset-criticality.html#bulk-assign-asset-criticality[bulk assign asset criticality] using the file upload feature, the newly assigned criticality levels are automatically factored in during the next hourly risk scoring calculation. You can now manually trigger an immediate recalculation of entity risk scores by clicking **Recalculate entity risk scores now** during the file upload process.
-
-[role="screenshot"]
-image::whats-new/images/8.16/recalc-ers.png[Recalculate entity risk scores]
-
 [float]
 == Detection rules and alerts enhancements
 
 [float]
-=== Enable prebuilt detection rules on installation
-
-Previously, {security-guide}/prebuilt-rules-management.html#load-prebuilt-rules[installing and enabling prebuilt rules] took two steps. You can now do both in one step with the **Install and enable** option. This works for both single and multiple rules.
+=== LogsDB index mode with detection rules and alerts
 
-[role="screenshot"]
-image::whats-new/images/8.16/install-enable-rules.png[Install and enable rules, 80%]
+The {ref}/logs-data-stream.html[logsDB index mode] allows you to store log data more efficiently. If you're considering using it, refer to <TBD> to learn how it can impact your rules and alerts. This feature requires the <TBD>.
```

Comment:
https://github.com/elastic/security-docs/pull/6272 needs to be merged for this reference to properly resolve. 

```suggestion
The {ref}/logs-data-stream.html[logsDB index mode] allows you to store log data more efficiently. If you're considering using it, refer to <<detections-logsdb-index-mode-impact>> to learn how it can impact your rules and alerts. 
```